pkg/cmd/roachtest/tests/admission_control_elastic_io.go

```diff
@@ -71,12 +71,17 @@ func registerElasticIO(r registry.Registry) {
 			duration := 30 * time.Minute
 			t.Status("running workload")
 			m := c.NewMonitor(ctx, c.CRDBNodes())
+			labels := map[string]string{
+				"duration":    fmt.Sprintf("%d", duration.Milliseconds()),
+				"concurrency": "512",
+			}
 			m.Go(func(ctx context.Context) error {
 				dur := " --duration=" + duration.String()
 				url := fmt.Sprintf(" {pgurl%s}", c.CRDBNodes())
-				cmd := "./cockroach workload run kv --init --histograms=perf/stats.json --concurrency=512 " +
-					"--splits=1000 --read-percent=0 --min-block-bytes=65536 --max-block-bytes=65536 " +
-					"--txn-qos=background --tolerate-errors --secure" + dur + url
+				cmd := fmt.Sprintf("./cockroach workload run kv --init %s -concurrency=512 "+
```

Comment:
`-concurrency` should be `--concurrency`

Reply:
@sambhav-jain-16 This broke https://github.com/cockroachdb/cockroach/issues/135393.